### ISSUE TITLE
Cloze Deletion answers unreadable colour, .cloze CSS selector broken

### ISSUE THREAD

**Issue by michael-angelozzi** [krassowski/Anki-Night-Mode#54]
#### Problem description
The close deletion answer is a dark blue on a black background. I cannot read it on my screen.
CSS selectors for cloze deletion answers not longer work with night mode installed, i.e.:

> .cloze, .nightMode .cloze { color: #00FF00 !important; }**

>  \* { color: #00FF00 !important; }  /*This styles everything EXCEPT the cloze deletion answers in night mode, in normal mode they are affected */**

#### Checklist

*Please put x inside brackets if following sentences apply to you:*

 - [x ] I verified that I use the lastest add-on version,
 - [ x] I verified that I use the lastest Anki version,
 - [ x] I tried to disable other add-ons to find the culprit,
 - [x ] I checked if anyone reported this problem before, as well as Known issues section on add-ons page,
 - [ ] My issue is solved when holding shift while starting Anki.
 
#### Information about your Anki Setup

*Please open Anki, go to Help → About, look-up two lines, starting with `Version:` and `Qt` and copy-paste them below:*

```
Version: Version 2.1.5
Qt: Qt 5.9.2 PyQt 5.9
```

*Please got to Night Mode → About now and copy-paste the first line from the pop-up window below:*

```
Night Mode x.x.x (uninstalled now but was the latest)
```

*Please fill in your OS details (Linux/Mac/Windows):*

```
OS: Windows 10
```
 
 
*If you have a compatibility problem with a specific add-on or add-ons, please paste the add-on page links below:*
 - [Other add-on name](https://)
</details>

#### Backtrace (if any)

<details>

*If you had an error which resulted in producing exception with a backtrace, please paste the backtrace below:*

```python"
[paste your backtrace in here if you got any; otherwise feel free to remove this part]
```

</details>

**Comment by krassowski** [comment]
Hi Michael,

let me explain why it does not work for you: the correct class name for this add-on (as described in the [README file](https://github.com/krassowski/Anki-Night-Mode#custom-css-in-night-mode)) is `.night_mode`, while you are using `.nightMode`.

Please, see the code and screenshot below: 

```css
.night_mode .cloze {
 color: red!important;
}
```

![screenshot from 2018-12-02 11-52-57](https://user-images.githubusercontent.com/5832902/49339380-db1c7480-f628-11e8-9058-79b9417d793d.png)

Also, if you used `.night_mode *` selector, this would work for you as well. In simpler terms cloze deletion is not broken - it only requires higher specificity, and you have incidentally used the wrong class name. Please let me know if everything works for you now. If you would like to lower the specificity required, please feel free to send PR, we can discuss the pros and cons then.
